shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/RewardUpdate.hs

```diff
@@ -210,11 +224,10 @@ data FreeVars crypto = FreeVars
     slotsPerEpoch :: !EpochSize,
     pp_d :: !UnitInterval, -- The last three fields come from some version of PParams
     pp_a0 :: !Rational,
-    pp_nOpt :: Natural
+    pp_nOpt :: !Natural
```

Comment:
This one, surprisingly, might also not be necessary, since I'm guessing we likely build `FreeVars` by passing it a forced value for `pp_nOpt`.

However, it's probably best if only consistency.

Reply:
yes, I confirmed with the threadnet tests that this one is not necessary. But, as you say, it is nice for consistency so I left it in.